challenge.md

```diff
@@ -6,19 +6,27 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
+'git clone (SSH link)'' to clone the repo into my computer
+
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
+'git diff' will show you all the changes since last commit, but not all the changes are necessarily followed unless 'git add (folder or file)' to move changes into the staging area.
```

Comment:
We were actually looking for `git status`, but based on how the question is phrased, I can see why you would jump to `git diff`, and won't take any points off. `git status` will show all modified files, though it won't show the specific changes, while `git diff` will show individual changes made to already-tracked files. You really want to use _both_ of them.